src/routes/+page.svelte

```diff
@@ -21,53 +17,118 @@
+
+        p {
+            text-align: center;
+            margin-bottom: 2rem;
+            color: #ffffff; /* White paragraph text color */
+        }
+
+        form {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            margin-bottom: 2rem;
+        }
+
+        label {
+            font-size: 1.5rem;
+            margin-bottom: 0.5rem;
+            color: #ffffff; /* White label text color */
+        }
+
+        #username-field {
+            padding: 0.5rem;
+            font-size: 1rem;
+            width: 300px;
+            border: 2px solid #ffffff; /* White border color */
+            border-radius: 4px;
+            font-family: 'Orbitron', sans-serif;
+            color: #ffffff; /* White text color */
+            background-color: rgba(0, 0, 0, 0.5); /* Semi-transparent black background */
+        }
+
+        .users {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            gap: 1rem;
+        }
+
+        .user {
+            width: 200px;
+            padding: 1rem;
+            border: none;
+            background: linear-gradient(45deg, rgba(72, 61, 139, 0.8), rgba(75, 0, 130, 0.8)); /* Shades of blue and purple */
+            color: #ffffff; /* White text color */
+            font-size: 1rem;
+            text-align: center;
+            border-radius: 4px;
+            cursor: pointer;
+            transition: background 0.3s ease, box-shadow 0.3s ease;
+            box-shadow: 0 0 15px rgba(75, 0, 130, 0.5); /* Purple glow */
+        }
+
+        .user:hover {
+            background: linear-gradient(45deg, rgba(72, 61, 139, 1), rgba(75, 0, 130, 1)); /* More intense shades on hover */
+            box-shadow: 0 0 20px rgba(75, 0, 130, 0.7); /* More intense glow on hover */
+        }
+    </style>
 </svelte:head>
 
-<h1>Vail stats</h1>
-<p>Delivered to you from far outside this world.</p>
-
-<h1>Search for user</h1>
-<form on:submit|preventDefault={() => {}}>
-    <label for="username-field">Username</label>
-    <br/>
-    <input id="username-field" type="search" placeholder={userPlaceholder} on:input={debounce(onSearchFieldChange)} bind:value={searchQuery}>
-</form>
-
-<div class="users">
-    {#each $searchResults as user}
-        <button class="user" on:click={() => {goto(`/players/${user.id}`)}}>
-            {user.name}
-        </button>
-    {/each}
+<div class="container">
+    <h1>Vail stats</h1>
+    <p>Delivered to you from far outside this world.</p>
+
+    <h1>Search for user</h1>
+    <form on:submit|preventDefault={() => {}}>
+        <label for="username-field">Username</label>
+        <input id="username-field" type="search" placeholder={userPlaceholder} on:input={debounce(onSearchFieldChange)} bind:value={searchQuery}>
+    </form>
+
+    <div class="users">
+        {#each $searchResults as user}
+            <button class="user" on:click={() => {goto(`/players/${user.id}`)}}>
```

Comment:
I wanted this to be a link but couldn't figure it out. Could you try?

Reply:
I will try it, What should it look like?

Reply:
The same. Want to change it so svelte optimises things

Reply:
So make it not have the user ID in the link?